src/ng/compile.js

```diff
@@ -1970,26 +1970,25 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
         }
         if (elementControllers) {
           // Initialize bindToController bindings for new/isolate scopes
-          var scopeDirective = newIsolateScopeDirective || newScopeDirective;
-          var bindings;
-          var controllerForBindings;
-          if (scopeDirective && elementControllers[scopeDirective.name]) {
-            bindings = scopeDirective.$$bindings.bindToController;
-            controller = elementControllers[scopeDirective.name];
-
-            if (controller && controller.identifier && bindings) {
-              controllerForBindings = controller;
-              thisLinkFn.$$destroyBindings =
-                  initializeDirectiveBindings(scope, attrs, controller.instance,
-                                              bindings, scopeDirective);
-            }
-          }
           for (i in elementControllers) {
+            var scopeDirective = newIsolateScopeDirective || controllerDirectives[i];
+            var bindings;
+            var controllerForBindings;
+            if (scopeDirective && elementControllers[scopeDirective.name]) {
```

Comment:
I think is probably not quite right any more, since `elementControllers[scopeDirective.name]` is not related to the current iteration of this for loop